src/seedsigner/gui/screens/scan_screens.py

```diff
@@ -18,7 +18,7 @@
 @dataclass
 class ScanScreen(BaseScreen):
     decoder: DecodeQR = None
-    instructions_text: str = "Scan a QR code"
+    instructions_text: str = "< back | Scan a QR code"
```

Comment:
Update to match seed image capture screen. Issue #185

Reply:
TY @EverydayBitcoiner, this improvement resolving an old issue has been borrowed in pr #323